pubspec.yaml

```diff
@@ -14,6 +14,8 @@ dependencies:
 
   cupertino_icons: ^1.0.2
   flutter_svg: ^2.0.9
+  auto_route: ^7.8.4
+  auto_route_generator: ^7.3.2
```

Comment:
I'm not sure, but auto_route_generator must by in dev_dependencies

Reply:
i made it with commands in Terminal, not with hands